release_dashboard/test/src/test_state_generator.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:conductor_core/conductor_core.dart';
+import 'package:conductor_core/proto.dart' as pb;
+
+const String conductorVersion = 'v1.0';
```

Comment:
These should all be moved as optional parameters in getTestState so they can be changed